snarkos/ledger/server.rs

```diff
@@ -196,14 +196,23 @@ impl<N: Network> Server<N> {
             .and(with(ledger_sender.clone()))
             .and_then(Self::transaction_broadcast);
 
-        // POST /testnet3/program/deploy
+        // POST /testnet3/deploy
         let deploy_program = warp::post()
             .and(warp::path!("testnet3" / "deploy"))
```

Comment:
Let's do "testnet3/program/deploy" and "testnet3/program/execute" when it's convenient to make the switch

Reply:
sorry do you mean keeping it like this or aligning the endpoint url with the old comment?

Reply:
I think we should align it to the old comments:

So the deploy would be "testnet3/program/deploy" and the execution would be "testnet3/program/execute"

Reply:
gotcha, I'll update and talk with the folks to adapt the clients of the endpoint accordingly

Reply:
updated